source/_patterns/30-organisms/masthead/masthead.twig

```diff
@@ -17,7 +17,9 @@
         </button>
       </div>
     </div>
-    <h1 class="masthead__title">News</h1>
+    <h1 class="masthead__title">
+        <a href="" class="masthead__title__link">News</a>
```

Comment:
Wie von Phil vorgeschlagen fände ich "masthead__title-link" auch besser.

Reply:
```suggestion
        <a href="" class="masthead__title-link">News</a>
```
Fänd ich auch besser

Reply:
Also der Link ist ja nicht zum masthead related, sondern zum title. .masthead__title-link wäre es doch, wenn ich es überall im masthead anwenden kann. Hier wird doch explizit gesagt, dass die Styles nur im masthead__title (der eigene Styles definiert) zur Anwendung kommen.
http://getbem.com/naming/

Reply:
Hoi!

Reply:
Paging the BEM-Boss. @pchr-srf

Reply:
.masthead__title-link

> BEM stands for Block__Element–Modifier, not Block__Element__Element–Modifier. So, avoid multiple element level naming. If you’re getting to great-great-great-grandchild levels, then you’ll probably want to revisit your component structure anyway.
> BEM naming isn’t strictly tied to the DOM, so it doesn’t matter how many levels deep a descendent element is nested. The naming convention is there to help you identify relationships with the top-level component block

https://www.smashingmagazine.com/2016/06/battling-bem-extended-edition-common-problems-and-how-to-avoid-them/

Reply:
Top! :)